src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,180 @@
+    *	The GraphQL resolvers for Query, Mutation or Subscription are converted into [VTL (Velocity Template Language)](https://docs.aws.amazon.com/appsync/latest/devguide/resolver-mapping-template-reference-programming-guide.html) files. AWS AppSync uses VTL to translate GraphQL requests from clients into a request to your data source.
+  * GraphQL schema Client-Side Code generation: The Amplify CLI GraphQL workflow allows developers to generate client side code for web and mobile clients using the amplify codegen command. 
+
+## Further Reading
+The [AWS Knowledge Center](https://aws.amazon.com/premiumsupport/knowledge-center/) provides a deep dive and prescriptive guidance for building and managing scalable. For debugging Amplify deployed services, the sections for AppSync, API Gateway, DynamoDB are highly recommended. 
+
+## Troubleshooting Amplify CLI Push Failures
+<Callout information>
+Error Messages: “An error occurred during the push operation”
+</Callout>
+
+The [`amplify push`](https://docs.amplify.aws/cli/start/workflows/#amplify-push) command performs the following steps before completing successfully such as the following in the case of a project using a GraphQL API:
+
+1. Generate CloudFormation for deployment of resources to AWS.
+2. Generate VTL & associated CloudFormation for the GraphQL API by running amplify api gql-compile. 
+3. Build packages and upload the deployment artifacts into the application's deployment bucket. 
+4. It then uses the AWS CloudFormation SDK to deploy the CloudFormation stack into the deployment account.
+5. On a successful deployment of the resources to AWS, the Amplify CLI copies the deployment package in the name #current-cloud-backend.zip. This file is the source of truth for your deployment. Hence even if your local folder gets deleted or corrupted, you can always go back to your “last successful deployment” using the [`amplify pull`](https://docs.amplify.aws/cli/start/workflows/#amplify-pull) command.
+###### fig.1.0 Amplify Folders created in the Deployment Bucket
+![ts-guide-amplify-deployment-folders](/images/ts-guide-amplify-deployment-folders.png)
+
+The following section shows some examples on how to debug `amplify push` failures: 
+
+### Handling Push Failures from Manual Changes to Cloud Resources - Drift 
+<Callout warning>
+Warning** 
+Once an application is deployed to the cloud using Amplify CLI the cloud resources like DynamoDB tables, Cognito Roles, IAM policies should never be updated manually from the AWS console of the services. 
```

Comment:
It's probably worth either calling out other ways besides the AWS console, or not calling out any specific ways to update resources. I'd just say don't do it.